app/src/test/java/org/fossasia/openevent/app/unit/presenter/AttendeeCheckInPresenterTest.java

```diff
@@ -124,7 +125,7 @@ public void shouldHandleAttendeeChange() {
         notifier.onNext(new DatabaseChangeListener.ModelChange<>(toggled, BaseModel.Action.UPDATE));
 
         verify(attendeeCheckInView).showResult(toggled);
-        verify(attendeeCheckInView).onSuccess(any());
+        verify(attendeeCheckInView, atLeast(2)).onSuccess(any());
```

Comment:
in onSuccess the `checkinAttendee` binding property is notified for change. So it will be called more than once.